verification/curator-service/ui/src/components/new-case-form-fields/Demographics.tsx

```diff
@@ -30,7 +32,13 @@ const styles = () =>
         },
     });
 
-type Props = WithStyles<typeof styles>;
+interface DemographicsProps extends WithStyles<typeof styles> {
+    setFieldValue: (
```

Comment:
just to check but formik doens't export a type for that?

Reply:
Not specifically, but I found a much cleaner way to get access to it through using useFormikContext(). Thanks for checking!